proxy-node-translator/src/main/java/uk/gov/ida/notification/translator/saml/HubResponseTranslator.java

```diff
@@ -115,18 +115,18 @@ private static String getMappedStatusCode(VspScenario vspScenario) {
     }
 
     private static String getCombineFirstAndMiddleNames(HubResponseContainer hubResponseContainer) {
-        var firstNames = hubResponseContainer.getAttributes().getFirstNames();
+        var firstNames = hubResponseContainer.getAttributes().getFirstNamesAttributesList();
```

Comment:
Wouldn't this cause a NPE since you're doing `this.attributes = translatedHubResponse.getAttributes().orElse(null)` in `HubResponseContainer`? I think the Optional should carry through to here and be handled safely at the point of usage. If the attributes are `Optional.empty` then in the `getTranslatedHubResponse` method above we can skip a bunch of stuff